server/src/shared/filters.ts

```diff
@@ -0,0 +1,106 @@
+import { FILTER_PARAM_APPROACH, FILTER_PARAM_BATTLEFIELD, FILTER_PARAM_COMBOS, FILTER_PARAM_CONGO, FILTER_PARAM_DK, FILTER_PARAM_DREAMLAND, FILTER_PARAM_EDGEGUARDING, FILTER_PARAM_ESCAPES, FILTER_PARAM_FALCON, FILTER_PARAM_FINAL_DESTINATION, FILTER_PARAM_FOX, FILTER_PARAM_HYRULE, FILTER_PARAM_JIGGLY, FILTER_PARAM_KIRBY, FILTER_PARAM_LINK, FILTER_PARAM_LUIGI, FILTER_PARAM_MARIO, FILTER_PARAM_META_CRYSTAL, FILTER_PARAM_MUSHROOM, FILTER_PARAM_NESS, FILTER_PARAM_PEACH, FILTER_PARAM_PIKA, FILTER_PARAM_SAFFRON, FILTER_PARAM_SAMUS, FILTER_PARAM_SECTOR_Z, FILTER_PARAM_YOSHI, FILTER_PARAM_YOSHI_ISLAND_19XX, FILTER_PARAM_ZEBES } from './query_params';
```

Comment:
I forgot to remove this in my last CL, but I don't think we need anything in this file on the server.

Reply:
Done